docs/sdk-and-tools/indices/scresults.md

```diff
@@ -0,0 +1,56 @@
+| receiver          | The receiver field represents the destination address of the smart contract result.                                                                            |
+| senderShard       | The senderShard field represents the shard ID of the sender address.                                                                                           |
+| receiverShard     | The receiverShard field represents the shard ID of the receiver address.                                                                                       |
+| relayerAddr       | The relayerAddr field represents the address of the relayer.                                                                                                   |
+| relayedValue      | This relayedValue field represents the amount of EGLD to be transferred via the inner transaction's sender.                                                    |
+| code              | The code holds the code of the smart contract result.                                                                                                          |
+| data              | The data field holds additional information for a smart contract result. It can contain a simple message, a function call, an ESDT transfer payload, or so on. |
+| prevTxHash        | The prevTxHash holds the hex encoded hash of the previous transaction.                                                                                         |
+| originalTxHash    | The originalTxHash holds the hex encoded hash of the transaction that generated the smart contract result.                                                     |
+| callType          | The callType field holds the type of smart contract call that is done through the smart contract result.                                                       |
+| codeMetaData      | The codeMetaData field holds the code metadata.                                                                                                                |
+| returnMessage     | The returnMessage field holds the message that is returned by a smart contract in case of an error.                                                            |
+| timestamp         | The timestamp field represents the timestamp of the block in which the smart contract result was executed.                                                     |
+| status            | The status field holds the execution state of the smart contract result. The execution state can be `pending` or `success`.                                    |
+| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field.                                                                  |
+| esdtValues        | The esdtValues field contains a list of ESDT values that are transferred based on the data field.                                                              |
```

Comment:
Can you also add here that the indexes of `tokens` are linked to indexes in `esdtValues` list?

Reply:
added